checkout/tests/test_get_checkout_summary.py

```diff
@@ -59,8 +59,8 @@ def test_get_delivery_options(client_buyer, cart):
     cart_item_0 = CartItem.objects.get(id=items[0]["id"])
     cart_item_1 = CartItem.objects.get(id=items[1]["id"])
 
-    assert cart_item_0.delivery_fee_gross == 32.73
-    assert cart_item_0.delivery_fee_net == 27.5
+    assert cart_item_0.delivery_fee_gross == 26.3
```

Comment:
We have to stop writing tests like this. I never know where these values come from and whether they are correct. 😢

Reply:
How we could do it better? This way: https://github.com/stanwood/traidoo-api/pull/79/files#diff-28921130bc3eeb55ab0954ea21dbc9c0R16

Reply:
Create a CSV/XLS file with lots of examples validated manually. We would need to read the values, use in unit tests and compare our results with data from the file.

Reply:
MCS-2045